src/lib/eina/eina_sched_win32.c

```diff
@@ -0,0 +1,64 @@
+/* EINA - EFL data type library
+ * Copyright (C) 2020 Carlos Signor
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library;
+ * if not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "eina_sched.h"
+#ifndef WIN32_LEAN_AND_MEAN
+# define WIN32_LEAN_AND_MEAN
+#endif
+#include <Windows.h>
+
+EAPI void eina_sched_prio_drop(void)
+{
+   Eina_Thread pthread_id;
```

Comment:
```suggestion
   Eina_Thread thread_id;
```
Maybe thread_id as we are not in a POSIX world.

Reply:
wtf